central/views/imagecve/view_test.go

```diff
@@ -78,6 +80,23 @@ func TestGetImageCVECore(t *testing.T) {
 		require.NoError(t, store.UpsertImage(ctx, image))
 	}
 
+	// Ensure that the image is stored and constructed as expected.
+	for idx, image := range images {
+		actual, found, err := store.GetImage(ctx, image.GetId())
+		require.NoError(t, err)
+		require.True(t, found)
+
+		cloned := actual.Clone()
+		// Adjust dynamic fields.
+		standardizeImages(image)
```

Comment:
nit : Can call standardizeImages once and pass both images to it. `standardizeImages(image, cloned)`

Reply:
I refactored this entire test in the next PR which also has your nit addressed.